infrastructure/modules/aks/providers.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "< 4.39.0"
+      version = "< 4.48.0"
```

This provider is locked to <4.39.0 due to a bug with diagnostics
```suggestion
      version = "< 4.39.0"
```